app/components/fixed-footer.tsx

```diff
@@ -8,7 +8,7 @@ export function FixedFooter({ children }: { children?: React.ReactNode }) {
   return (
     <footer
       id="footer"
```

Comment:
The footer was overlapping content. Width zero works nicely here to prevent that.